msteams-platform/toolkit/TeamsFx-CLI.md

```diff
@@ -1,23 +1,390 @@
+
+* Show permission status
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view Microsoft 365 account permission for specific env:
+
+  ```typescript
+  teamsapp permission status --env dev
+  ```
+
+* List all collaborators
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view all collaborators for specific env:
+
+  ```typescript
+  teamsapp collaborator status --env dev --list-all-collaborators
+  ```
+
+## `teamsapp update aad-app`
+
+Update the specific application manifest file.
+
+| Command | Description |
+|:----------------  |:-------------|
+| `teamsapp entra-app update` | Update the Microsoft Entra ID app in the current application. |
+| `teamsapp update` | Update the Teams app manifest to Teams Developer Portal. |
+
+### Parameters for `teamsapp entra-app update`
+
+| Parameter | Required |Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--folder` | No | Select root folder of the project. Defaults to `./` |
+| `--entra-app-manifest-file` | No | Enter the Microsoft Entra ID app manifest template file path, it's a relative path to project root folder, defaults to `./aad.manifest.json`. |
+
+### Parameters for `teamsapp update teams-app`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Specifies the environment name for the project scaffolded by Microsoft Teams Toolkit. |
+| `--env-file` | Yes | Specifies the .env file that defines the env variables to replace in the Teams app manifest template file. |
+| `--folder` | No | Select root folder of the project. Defaults to `./` |
+| `--manifest-file` | No | Specifies the Microsoft Teams app manifest file path. Default value: `./appPackage/manifest.json`. |
+| `--package-file` |  | Specifies the zipped Microsoft Teams app package file path. |
+| `--output-package-file` |  | Specifies the output zipped Microsoft Teams app package file path. Default value: `./appPackage/build/appPackage.${env}.zip`. |
+| `--output-manifest-file` |  | Specifies the final output Microsoft Teams app manifest file path in which the placeholders are resolved. |
+
+## `teamsapp upgrade`
+
+Upgrade the project to work with the latest version of Teams Toolkit.
+
+### Parameters for `teamsapp upgrade`
+
+|Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--force` | No | Force upgrade the project to work with the latest version of Teams Toolkit. Defaults to `false`. |
+
+## `teamsapp doctor`
+
+Prerequisites to create Teams application.
+
+| Command | Description |
+|:----------------  |:-------------|
+| `teamsapp install` | Upload a custom app for a given application package across Microsoft 365. |
```

Comment:
These three commands are not prerequisites of `teamsapp doctor`.They are indepedent commands for m365 app.

Reply:
Updated.